### PR TITLE
On agnomina and localization

### PR DESCRIPTION
**Pull request by IohannesIohannium**
- Moved ir_names into correct folders
- Provided Spanish, Chinese and Korean missing locs (although done with Google Translator)
- Added more nicknames to support all I:R ones except for regnal numbers
- Removed nameless "dynn_" and "gens_" from a couple culture files

- Added Holy Sites localization
- Corrected location of a couple Holy Sites

### CODE REVIEW

**Comment by IhateTrains** [comment]
Do you intend to add more changes, or is it ready for review?

**Comment by IohannesIohannium** [comment]
It is ready for review.

File: ImperatorToCK3/Data_Files/configurables/nickname_map.txt [hunk 1]
```diff
@@ -31,50 +31,50 @@ link = { imp = NICKNAME_CURIO imp = NICKNAME_THE_DEVOUT ck3 = nick_the_pious }
...
-#link = { imp = NICKNAME_GALLICUS imp = NICKNAME_THE_GAUL ck3 = ??? } #"The French"
-#link = { imp = NICKNAME_BELGICUS imp = NICKNAME_THE_BELGAE ck3 = ??? } #"The Belgian"
+link = { imp = NICKNAME_BRITANNICUS imp = NICKNAME_THE_BRITON ck3 = nick_the_british }
+link = { imp = NICKNAME_GALLICUS imp = NICKNAME_THE_GAUL ck3 = nick_the_french }
+link = { imp = NICKNAME_BELGICUS imp = NICKNAME_THE_BELGAE ck3 = nick_the_belgian }
 link = { imp = NICKNAME_GERMANICUS imp = NICKNAME_THE_GERMAN ck3 = nick_the_german }
-#link = { imp = NICKNAME_RHAETICUS imp = NICKNAME_THE_RHAETIAN ck3 = ??? } #"The Rhetian"
-#link = { imp = NICKNAME_ITALICUS imp = NICKNAME_THE_ITALIOTE ck3 = ??? } #'The Italiote'
-#link = { imp = NICKNAME_ILLYRICUS imp = NICKNAME_THE_ILLYRIAN ck3 = ??? } #'The Illyrian'
+link = { imp = NICKNAME_RHAETICUS imp = NICKNAME_THE_RHAETIAN ck3 = nick_the_swiss }
```
Comment (IhateTrains) [inline ImperatorToCK3/Data_Files/configurables/nickname_map.txt:38]: Why not keep it as "the Rhaetian"?

File: ImperatorToCK3/Data_Files/configurables/nickname_map.txt [hunk 2]
```diff
@@ -31,50 +31,50 @@ link = { imp = NICKNAME_CURIO imp = NICKNAME_THE_DEVOUT ck3 = nick_the_pious }
...
-#link = { imp = NICKNAME_MAURETANICUS imp = NICKNAME_THE_MAURETANIAN ck3 = ??? } #'The Mauretanian'
-#link = { imp = NICKNAME_NUMIDIUS imp = NICKNAME_THE_NUMIDIAN ck3 = ??? } #'The Numidian'
+link = { imp = NICKNAME_MAURETANICUS imp = NICKNAME_THE_MAURETANIAN ck3 = nick_the_moor }
+link = { imp = NICKNAME_NUMIDIUS imp = NICKNAME_THE_NUMIDIAN ck3 = nick_the_algerian }
 link = { imp = NICKNAME_VENEDICUS imp = NICKNAME_THE_VENEDIAN ck3 = nick_the_wend }
-#link = { imp = NICKNAME_DACIUS imp = NICKNAME_THE_DACIAN ck3 = ??? } #'The Dacian'
-#link = { imp = NICKNAME_HELLENICUS imp = NICKNAME_THE_HELLENE ck3 = ??? } #'The Hellene'
-#link = { imp = NICKNAME_CYRENAICUS imp = NICKNAME_THE_CYRENE ck3 = ??? } #'The Cyrene'
-#link = { imp = NICKNAME_AEGYPTUS imp = NICKNAME_THE_EGYPTIAN ck3 = ??? } #'The Egyptian'
+link = { imp = NICKNAME_DACIUS imp = NICKNAME_THE_DACIAN ck3 = nick_the_romanian }
```
Comment (IhateTrains) [inline ImperatorToCK3/Data_Files/configurables/nickname_map.txt:45]: Why not just Dacian? Only a Romanized Dacian would be called Romanian..

File: ImperatorToCK3/Data_Files/configurables/nickname_map.txt [hunk 3]
```diff
@@ -31,50 +31,50 @@ link = { imp = NICKNAME_CURIO imp = NICKNAME_THE_DEVOUT ck3 = nick_the_pious }
...
-#link = { imp = NICKNAME_SCYTHIANUS imp = NICKNAME_THE_SCYTHIAN ck3 = ??? } #'The Scythian'
-#link = { imp = NICKNAME_PERSICUS imp = NICKNAME_THE_PERSIAN ck3 = ??? } #'The Persian'
-#link = { imp = NICKNAME_BACTRIANUS imp = NICKNAME_THE_BACTRIAN ck3 = ??? } #'The Bactrian'
-#link = { imp = NICKNAME_GEDROSIANUS imp = NICKNAME_THE_GEDROSIAN ck3 = ??? } #'The Gedrosian'
-#link = { imp = NICKNAME_INDICUS imp = NICKNAME_THE_INDIAN ck3 = ??? } #'The Indian'
-#link = { imp = NICKNAME_DRAVIDICUS imp = NICKNAME_THE_DRAVIDIAN ck3 = ??? } #'The Dravidian'
+link = { imp = NICKNAME_ARABICUS imp = NICKNAME_THE_ARABIAN ck3 = nick_the_arabian }
+link = { imp = NICKNAME_SYRIANUS imp = NICKNAME_THE_SYRIAN ck3 = nick_the_syrian }
+link = { imp = NICKNAME_ASIATICUS imp = NICKNAME_THE_ASIAN ck3 = nick_the_asian }
+link = { imp = NICKNAME_SARMATICUS imp = NICKNAME_THE_SARMATIAN ck3 = nick_the_russian }
```
Comment (IhateTrains) [inline ImperatorToCK3/Data_Files/configurables/nickname_map.txt:53]: Sarmatians were Iranian peoples, so does it make sense to convert to "the Russian"?

**Review by IhateTrains — APPROVED** [review]

File: ImperatorToCK3/Data_Files/configurables/nickname_map.txt (hunk 1 above)
Comment (IohannesIohannium) [reply ImperatorToCK3/Data_Files/configurables/nickname_map.txt:38]: Solved through localization. nick_the_swiss, nick_the_romanian, nick_the_russian actually convert to the Rhaetian, the Dacian, the Sarmatian.

File: ImperatorToCK3/Data_Files/configurables/nickname_map.txt (hunk 3 above)
Comment (IohannesIohannium) [reply ImperatorToCK3/Data_Files/configurables/nickname_map.txt:53]: Ditto

File: ImperatorToCK3/Data_Files/configurables/nickname_map.txt (hunk 2 above)
Comment (IohannesIohannium) [reply ImperatorToCK3/Data_Files/configurables/nickname_map.txt:45]: Ditto

**Pull request merged**